DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
I did read your comment:  "which will eventually be replaced by the eras.run2", but is this true ?  Why do we still see the CSCTF DQM plots with run 1 configuration if it will eventually be replaced by the eras.run2 ?